datadog/resource_datadog_service_level_objective.go

```diff
@@ -94,21 +94,21 @@ func resourceDatadogServiceLevelObjective() *schema.Resource {
 				},
 			},
 			"target_threshold": {
-				Description:      "The objective's target in `(0,100)`.",
+				Description:      "The objective's target in `(0,100)`. This must match the corresponding thresholds of the primary time frame ",
 				Type:             schema.TypeFloat,
 				Optional:         true,
 				Computed:         true,
 				DiffSuppressFunc: suppressDataDogFloatIntDiff,
 			},
 			"warning_threshold": {
-				Description:      "The objective's warning value in `(0,100)`. This must be greater than the target value.",
+				Description:      "The objective's warning value in `(0,100)`. This must be greater than the target value, and match the corresponding thresholds of the primary time frame.",
```

Comment:
```suggestion
				Description:      "The objective's warning value in `(0,100)`. This must be greater than the target value and match the corresponding thresholds of the primary time frame.",
```

Reply:
Fixed in [f421d46](https://github.com/DataDog/terraform-provider-datadog/pull/1774/commits/f421d461db0f71dfc3420048e70cf560ee979b8d)